src/main/json/r6OperatorCounters.json

```diff
@@ -1485,6 +1485,15 @@
                     "properties": {
                         "Description": "The Thermal Feedback System on Glaz's OTs-03 can see through Smoke's Gas Grenades."
                     }
+                },
+                {
+                    "id": "141",

```

Comment:
I just merged in a different PR adding ID 141. Sorry for the trouble but could you update yours to 142, please?

Reply:
You might need to do a merge from master or rebase on top of master. You will see this conflict then and can update to 142.

Reply:
There still looks to be conflicts on this branch. https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github